pkg/kubelet/kuberuntime/kuberuntime_container_test.go

```diff
@@ -506,6 +507,8 @@ func TestToKubeContainerStatusWithUser(t *testing.T) {
 		},
 	} {
 		t.Run(desc, func(t *testing.T) {
+			// Set emulation version so that the feature gate can be disabled in the test
+			featuregatetesting.SetFeatureGateEmulationVersionDuringTest(t, utilfeature.DefaultFeatureGate, version.MustParse("1.33"))
```

Comment:
Why 33?  Isn't it usually the last version where the gate was mutable?

Reply:
Thanks for the catch, too! I'll update it with 1.34.

> Why 33?

I chose 33 because it was the beta.